frontend/apps/reader/modules/readerpaging.lua

```diff
@@ -396,8 +405,13 @@ function ReaderPaging:onPan(_, ges)
         if self.ui.gesture and self.ui.gesture.multiswipes_enabled then
             relative_type = "relative_delayed"
         end
-        self:onPanningRel(self.last_pan_relative_y - ges[relative_type].y)
-        self.last_pan_relative_y = ges[relative_type].y
+        --this is only use when mouse wheel is used
```

Comment:
Sorry, I'm just slightly distracted by it. :-P
```suggestion
        -- this is only used when mouse wheel is used
```